pages/decentralized-infrastructure/subgraphs/record.mdx

```diff
@@ -0,0 +1,37 @@
+# Record
+
+##### Record
```

Comment:
I think we may want to not jump heading levels here -- maybe we could just bold it?

No big deal, we can review and check this out after the fact